### PR TITLE
Replace deprecated package 'react-query' with '@tanstack/react-query'

### PR DESCRIPTION
**Pull request by alex-ahumada**
Updated Marketing and Umami examples.

### CODE REVIEW

**Comment by shadcn** [comment]
Oh woah! I haven't even looked at the upgrade docs yet. Appreciate it. Thank you.

**Comment by shadcn** [comment]
I'll take a look at the failing tests (which is not related to the upgrade). Thank you.

**Comment by alex-ahumada** [comment]
Thanks @shadcn ! We're building a new project we'll release later this year using next-drupal.
We have a working POC using Layout Builder instead of Paragraphs. We still need to smooth out some rough edges but I hope I can contribute it to the project soon 😄 
Thanks for this awesome tool!

File: examples/example-marketing/package.json [hunk 1]
```diff
@@ -25,7 +26,6 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-hook-form": "^7.8.6",
```
Comment (shadcn) [inline examples/example-marketing/package.json:28]: @alex-ahumada Minor fix here. We have an extra trailing comma.

Comment (alex-ahumada) [reply examples/example-marketing/package.json]: Done!

**Pull request merged**